CMakeLists.txt

```
@@ -36,6 +36,13 @@ option(BUILD_SHARED_LIBS "Build the shared library (instead of a static lib)" TR
 option(NEW_HOST_WRAPPER "Use the new host wrapper" TRUE)
 option(SF2_SUPPORT "Enable SF2 support and float DCF" TRUE)
 option(ZLIB_SUPPORT "Enable XMF ZLIB Unpacker support" TRUE)
+option(EAS_WT_SYNTH "Enable WaveTable Synth and build its sound library" TRUE)
+option(EAS_FM_SYNTH "Enable FM Synth and build its sound library" FALSE)
```

Sorry, but I don't see the point of repeating "and build its sound library" here. What would be the meaning of "Enable" that needs so many words to clarify?

```suggestion
option(EAS_WT_SYNTH "Enable WaveTable Synth" TRUE)
option(EAS_FM_SYNTH "Enable FM Synth" FALSE)
```